packages/react-devtools-shell/README.md

```diff
@@ -2,7 +2,7 @@ Harness for testing local changes to the `react-devtools-inline` and `react-devt
 
 ## Development
 
-This target should be run in parallel with the `react-devtools-inline` package. The first step then is to run that target following the instructions in the [`react-devtools-inline` README's local development section](https://github.com/facebook/react/tree/main/packages/react-devtools-inline#local-development).
+This target should be run in parallel with the `react-devtools-inline` package. The first step then is to run that target following the instructions in the [`react-devtools-inline` README's local development section](../react-devtools-inline/README.md#local-development).
```

Comment:
drive-by fix. Links to the local README in vscode and works on GitHub still: https://github.com/eps1lon/react/blob/sebbie/04-22-_devtools_allow_inspection_of_used_thenables/packages/react-devtools-shell/README.md